kaldb/src/main/proto/metadata.proto

```diff
@@ -109,3 +109,28 @@ message RecoveryTaskMetadata {
   // Created timestamp
   int64 created_time_epoch_ms = 5;
 }
+
+message ServiceMetadata {
+  // Unique name for provisioned service
+  string name = 1;
+
+  // Service owner information (ie team name)
+  string owner = 2;
+
+  // Service throughput max
+  int64 throughput_bytes = 3;
+
+  // List of configured partitions
+  repeated ServicePartitionMetadata partition_configs = 4;
+}
+
+message ServicePartitionMetadata {
```

Comment:
Please add a comment saying that this struct contains the partition allocation for a service from  `start_time_epoch_ms` to `end_time_epoch_ms`.

Since this is an internal struct in ServiceMetadata, does this message need to be a metadata? It's logic can be added to ServiceMetadata alone. No?

Reply:
Comment added to top-level message. Discussion about using an internal struct being addressed in https://github.com/slackhq/kaldb/pull/236/files#r817925357